app/views/contexts/summary/block.py

```diff
@@ -111,10 +116,23 @@ def get_question(
             language=language,
         ).serialize()
 
-    def serialize(self) -> dict[str, Any]:
-        return {
-            "id": self.id,
-            "title": self.title,
-            "number": self.number,
-            "question": self.question,
-        }
+    def _handle_id_suffixing(self, block: dict) -> dict:
+        """
+        If the block is repeating, the block id, as well as any other ids (e.g. question, answer) need updating to be suffixed with list_item_id
```

Comment:
Worth being explicit, such as `If the block a list collector repeating block then ...`. We want to avoid confusion with repeating inside a repeating section.

Reply:
Updated as per suggestion